source/common/event/dispatcher_impl.h

```diff
@@ -30,7 +30,7 @@ class DispatcherImpl : Logger::Loggable<Logger::Id::main>, public Dispatcher {
   /**
    * @return event_base& the libevent base.
    */
-  event_base& base() { return *base_; }
+  event_base& base() const { return *base_; }
```

Comment:
Can you revert this const change also and a couple of the other ones you added?

Reply:
I will remove them but I think they should be const :)